Packs/QualysVulnerabilityManagement/Playbooks/playbook-Vulnerability_Management__-_Qualys_(Job)_V2.yml

```diff
@@ -0,0 +1,494 @@
+    skipunavailable: false
+    quietmode: 0
+  "11":
+    id: "11"
+    taskid: 73053224-7dd1-481b-80ec-e1cd452d9f22
+    type: regular
+    task:
+      id: 73053224-7dd1-481b-80ec-e1cd452d9f22
+      version: -1
+      name: Get report
+      description: Download report
+      script: QualysVulnerabilityManagementV2|||qualys-report-fetch
+      type: regular
+      iscommand: true
+      brand: QualysVulnerabilityManagementV2
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      file_format:
+        simple: xml
+      id:
+        complex:
+          root: QualysReport
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: QualysReport.TITLE
+                iscontext: true
+              right:
+                value:
+                  simple: inputs.QualysReportTitle
+                iscontext: true
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: QualysReport.STATUS.STATE
+                iscontext: true
+              right:
+                value:
+                  simple: Finished
+              ignorecase: true
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: QualysReport.OUTPUT_FORMAT
+                iscontext: true
+              right:
+                value:
+                  simple: XML
+              ignorecase: true
+          accessor: ID
+          transformers:
+          - operator: atIndex
+            args:
+              index:
+                value:
+                  simple: "0"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "12":
+    id: "12"
+    taskid: be76cf2f-e175-4386-8a66-61f413904f0a
+    type: regular
+    task:
+      id: be76cf2f-e175-4386-8a66-61f413904f0a
+      version: -1
+      name: Set context
+      description: 'Set the Qualys reports list into context. '
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      key:
+        simple: QualysReport
+      value:
+        complex:
+          root: Qualys
+          accessor: Report
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 690
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "14":
+    id: "14"
+    taskid: 76b7243d-96b0-4cd1-8f30-13015382d460
+    type: regular
+    task:
+      id: 76b7243d-96b0-4cd1-8f30-13015382d460
+      version: -1
+      name: Close Investigation
+      description: Close the investigation.
+      script: Builtin|||closeInvestigation
```

Comment:
@idovandijk @Arsenikr 
what is our best practice for this case?
the playbook is triggered by a job and the integration instance is not enabled.
if we close the investigation it might seem like the playbook finished successfully and continue to run the job without doing anything.
in my opinion, we should fail using `PrintError` - won't run the job again (depends on job configuration) and will attract the user's attention.

Reply:
It depends on what they want to do with incidents that fail. If they're OK with the job not doing it's job, then they can close. If they want to do something about it - yeah should fail it. They could also assign someone before failing.

Reply:
This playbook was copied from V1. This is how it has been used.

Reply:
let's change it please

Reply:
Fixed.